R/test_result.R

```diff
@@ -40,12 +42,13 @@ test_result <- function(
   grader_args = list(), # provided by `grade_learnr`
   learnr_args = list() # provided by `grade_learnr`
 ) {
-  chkm8_item_class(tests, "grader_test")
+  tests <- tests(...)
```

Comment:
let's change `tests` function to be `grader_tests` to avoid confusion for us. (and to match the S3 name.)